mhkit/wave/graphics.py

```diff
@@ -294,3 +294,49 @@ def plot_chakrabarti(H, lambda_w, D, ax=None):
     ax.set_ylabel('KC parameter, $\\frac{H}{D}$')
 
     plt.tight_layout()
+
+def plot_environmental_contour(x1, x2, x1_contour, x2_contour, 
```

Comment:
I would like to see this function be b able to plot multiple contours for different years simultaneously (using the direct output from environmental_contour) with each contour a different color, and with each contour having a different label. I am not sure if that change would be best made here or in the hidden function located in river.graphics that this is calling.

Reply:
Feel free to add that functionality.

Reply:
Wrapping multiple calls is something I would add in the next stage where we start bootstrapping the contours. See the WDRT environmental sampling example for a visual of this. This is also where the full functionality of passing a PCA becomes useful.